gammapy/irf/background.py

```diff
@@ -134,6 +134,31 @@ def to_fits(self, name='BACKGROUND'):
         """Convert to `~astropy.io.fits.BinTable`."""
         return fits.BinTableHDU(self.to_table(), name=name)
 
+    def evaluate(self, detx, dety, energy_reco, method=None, **kwargs):
+        """
+        Evaluate the `Background3D` at a given FOV coordinate and energy. The coordinates det_x, det_y and erngy_reco
+        should have the same dimension that match the numbers of point you want to evaluate
+        Parameters
+        ----------
+        detx : `~astropy.coordinates.Angle`
+                FOV coordinate X-axis binning. Same dimension than det_y and energy_reco
+        dety: `~astropy.coordinates.Angle`
+                FOV coordinate Y-axis binning. Same dimension than det_x and energy_reco
+        energy_reco : `~astropy.units.Quantity`
```

Comment:
You might use names fov_lon and fov_lat to comply with  https://github.com/gammapy/gammapy/blob/f91338a966a6c935f6ae66f36de2c838254b487a/docs/development/howto.rst

Reply:
I'll do this before merging.